wiki/en/Software-Manual.md

```diff
@@ -77,16 +77,24 @@ open automatically for all Clients. See Settings to optionally turn on a sound a
 
 Opens a dialogue where you can select a server to connect to. If you are connected,
 pressing this button will end the session.
+You can also open the dialogue using "File&gt;Connection Setup..."  This will leave you connected to any current session until you join another.
```

Comment:
```suggestion
You can also open the dialogue using "File&gt;Connection Setup...". This will leave you connected to any current session until you join another.
```

Reply:
Good spot.  (I even put the quotes so I could end the sentence cleanly!)